README.md

```diff
@@ -1,60 +1,33 @@
-# PSR-7 - Http Message Interface
+# Contributte PSR-7 - Http Message Interface
 
-:dizzy: PSR #7 [HTTP Message Interface] to Nette Framework.
-
------
+[PSR-7 HTTP Message Interface](https://www.php-fig.org/psr/psr-7/) to [Nette Framework](https://nette.org), based on [guzzle/psr-7](https://github.com/guzzle/psr7).
 
 [![Build Status](https://img.shields.io/travis/contributte/psr7-http-message.svg?style=flat-square)](https://travis-ci.org/contributte/psr7-http-message)
 [![Code coverage](https://img.shields.io/coveralls/contributte/psr7-http-message.svg?style=flat-square)](https://coveralls.io/r/contributte/psr7-http-message)
 [![Licence](https://img.shields.io/packagist/l/contributte/psr7-http-message.svg?style=flat-square)](https://packagist.org/packages/contributte/psr7-http-message)
 [![Downloads this Month](https://img.shields.io/packagist/dm/contributte/psr7-http-message.svg?style=flat-square)](https://packagist.org/packages/contributte/psr7-http-message)
 [![Downloads total](https://img.shields.io/packagist/dt/contributte/psr7-http-message.svg?style=flat-square)](https://packagist.org/packages/contributte/psr7-http-message)
 [![Latest stable](https://img.shields.io/packagist/v/contributte/psr7-http-message.svg?style=flat-square)](https://packagist.org/packages/contributte/psr7-http-message)
-[![PHPStan](https://img.shields.io/badge/PHPStan-enabled-brightgreen.svg?style=flat)](https://github.com/phpstan/phpstan)
+[![PHPStan](https://img.shields.io/badge/PHPStan-enabled-brightgreen.svg?style=flat-square)](https://github.com/phpstan/phpstan)
 
 ## Discussion / Help
 
 [![Join the chat](https://img.shields.io/gitter/room/contributte/contributte.svg?style=flat-square)](http://bit.ly/ctteg)
 
-## Install
+## Documentation
 
-```
-composer require contributte/psr7-http-message
-```
+- [Setup](.docs/README.md#setup)
+- [Psr7Request](.docs/README.md#psr7request)
+- [Psr7Response](.docs/README.md#psr7response)
 
 ## Versions
 
-| State       | Version   | Branch   | PHP      |
-|-------------|-----------|----------|----------|
-| development | `^0.6.0`  | `master` | `>= 7.1` |
-| stable      | `^0.5.0`  | `master` | `>= 7.1` |
-| stable      | `^0.3.0`  | `master` | `>= 5.6` |
-
-## Prolog
-
-Middleware / Relay pattern is widely used for handling any HTTP requests, such as API request, streams, dynamic websites 
```

Comment:
Where is this text in the new version?

Reply:
Nowhere. It's related to psr-7 middlewares, which were completely removed from psr-7 specs. Also, this description is in contributte/middlewares which implement this standard